src/compiler/compile-props.js

```diff
@@ -120,6 +125,28 @@ module.exports = function compileProps (el, propOptions) {
 }
 
 /**
+ * Check existance of an attribute with binding syntax.
+ *
+ * @param {Element} el
+ * @return {String} attr
+ */
+
+function checkBindingAttr (el, attr) {
+  if (attr === 'class') {
+    return null
+  }
+
+  return (
+    el.getAttribute(':' + attr) ||
+    el.getAttribute(':' + attr + '.once') ||
+    el.getAttribute(':' + attr + '.sync') ||
```

Comment:
Is the check for `.once` and `.sync` necessary? Does IE also treat `:title.once` as `title`?